docs/3_warehouse.md

```diff
@@ -13,14 +13,6 @@ The warehouse utility allows for the automation of complex nested workflows with
       python setup.py install
       python setup.py clean
    ```
-
```

Comment:
We shouldn't do this anymore due to lack of env reproducibility and the process being tedious.

`esgcet` should be regularly released with updates on Anaconda, which we would pull in through the `dev.yml`.

Reply:
Fine with me.  All I ever do (know HOW to do) is "python setup.py install", and then expect everything to work like magic.

As a "dumb user/developer", what should my typical procedures be, given this yml-driven process?

Regarding e3sm::zstash=1.1.0, what happens when it (zstash) gets updated?  Must the yml be updated manually?

Reply:
> As a "dumb user/developer", what should my typical procedures be, given this yml-driven process?

This document has the [Getting Started](https://github.com/E3SM-Project/esgfpub/blob/master/docs/1_developers_guide.md#getting-started) for setting up the dev env with the `.yml`. However, I think you're referring to if you already set up the dev env using the `.yml` and you want to update it with the latest versions of the dependencies.

If that's the case, we have two options:
1. Manually update the `.yml` with intended versions and merge it to `master`
2. Drop version pins for the dependencies where we always want the latest version
   - I'm assuming this applies to `zstash`, `esgcet`, and `autocurator` 

Then, update the dev env using the `.yml`: `conda env update -n esgfpub_dev -f conda/dev.yml`  

> Regarding e3sm::zstash=1.1.0, what happens when it (zstash) gets updated? Must the yml be updated manually?

Yes, we will have to update the `.yml` to get the latest `zstash` (option 1). But we can go with option 2 and remove the pinning.